### DISCUSSION TITLE
ltsp.conf   Restrict all clients to thin clients

### DISCUSSION THREAD

**Discussion by tichzvidzayi** [ltsp/ltsp#351] [Q&A]
How do you restrict all clients to thin clients, I have tried to use the line : 
                LTSP_FATCLIENT=False    
as with the previous version but it is not working.

**Comment by alkisg** [comment]
Hello @tzvidzie,

the new LTSP doesn't have a "thin client mode", please read https://github.com/ltsp/ltsp/discussions/243.
You can use any remote desktop technology that you want, like X -query, X2go, RDP, VNC etc though.

**Reply by tichzvidzayi** [reply]
thanks for the reply
